packages/common/src/Utility.ts

```diff
@@ -736,11 +736,17 @@ export function textSize(_text: string | string[], fontName: string = "Verdana",
     const hash = `${bold}::${fontSize}::${fontName}::${text.join("::")}`;
     let retVal = g_fontSizeContextCache[hash];
     if (!retVal) {
+        let dimensions: TextSize = { width: 0, height: 0 };
```

Comment:
Change to `retVal = { width: 0, height: 0 };`
And then remove dimension below...